bindings/nodejs/examples/client/14-build-foundry-output.ts

```diff
@@ -33,9 +33,12 @@ async function run() {
             '0xff311f59790ccb85343a36fbac2f06d233734794404142b308c13f2c616935b5';
 
         const foundryOutput = await client.buildFoundryOutput({
-            serialNumber: 0,
-            // 10 hex encoded
-            tokenScheme: new SimpleTokenScheme('0xa', '0x0', '0xa'),
+            serialNumber: 1,
+            tokenScheme: new SimpleTokenScheme(
+                BigInt(10),
+                BigInt(0),
+                BigInt(10),
+            ),
             amount: '1000000',
```

Comment:
Shouldn't this also be a BigInt now?

Reply:
I thought only native tokens would be Bigint?

Reply:
No, I think we want all Bigint, so it's easier to use (you can use the + * - % ** operators which will not work properly with strings)
Only native tokens get hex encoded when serialized, amount gets converted to a string

Reply:
done

Reply:
done